docker/docker-compose.yml

```diff
@@ -69,6 +69,34 @@ services:
       - 'traefik.http.middlewares.strip_auth_prefix.stripprefix.forceSlash=true'
       - 'traefik.http.routers.auth.middlewares=strip_auth_prefix'
 
+  # search service
+  search:
+    container_name: 'search'
+    build:
+      context: ../src/api/search
+      dockerfile: Dockerfile
+    environment:
```

Comment:
Inside this environemnt, you need to make sure all the `process.env` values you use are defined here, for example:

- ELASTIC_MAX_RESULTS_PER_PAGE
- ELASTIC_URL
- ELASTIC_PORT

I'm not sure if there are more?

Reply:
In `production.yml` you would override this:


      - ELASTIC_PORT=9200
      - ELASTIC_URL=http://elasticsearch

Reply:
In `development.yml` you would do:

```
ELASTIC_URL=http://localhost
```